cmd/scollector/main.go

```diff
@@ -149,7 +150,7 @@ func readConf() {
 
 func main() {
 	flag.Parse()
-	if *flagPrint || *flagDebug {
+	if *flagPrint || *flagDebug || *flagHTTP {
 		slog.Set(&slog.StdLog{Log: log.New(os.Stdout, "", log.LstdFlags)})
```

Comment:
I don't think http mode should setup logging this way. This is another breaking change to the default behaviour.